source/data/texture/TextureLoader.cpp

```diff
@@ -1,9 +1,8 @@
 #include <radix/data/texture/TextureLoader.hpp>
 
-#include <epoxy/gl.h>
-
+#include <radix/OpenGL.hpp>
 #include <radix/env/Environment.hpp>
-#include <FreeImagePlus.h>
+#include <FreeImage.h>
```

Comment:
afaik we are using freeimageplus.h, but i will try to build & run it only with freeimage.h